src/OpenTelemetry.Extensions.PersistentStorage.Abstractions/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog - OpenTelemetry.Contrib.Extensions.PersistentStorage
```

Comment:
nit: `# Changelog - OpenTelemetry.Contrib.Extensions.PersistentStorage.Abstractions`

Reply:
Just calling out loud - do we need to put the package name here at all? E.g. in opentelemetry-dotnet repo we don't do this and seems nobody is complaining https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/CHANGELOG.md.

Reply:
@reyang You are right - we do not need to. I will remove it.